WORKSPACE

```diff
@@ -70,6 +70,19 @@ http_archive(
     ],
 )
 
+# For development, one often wants to make changes to the OpenXLA repository as well
```

Comment:
why is this moved?

Reply:
local_repository needs to be stated before `load("@xla//third_party/py:python_init_rules.bzl", "python_init_rules")...` etc. Those are newly added from hermetic python.